src/accumulator/stump.rs

```diff
@@ -445,3 +445,56 @@ mod test {
         }
     }
 }
+#[cfg(bench)]
+mod bench {
+    extern crate test;
+    use super::Stump;
+    use crate::accumulator::proof::Proof;
+    use bitcoin_hashes::hex::FromHex;
+    use bitcoin_hashes::{sha256, Hash, HashEngine};
+    use test::Bencher;
+    #[bench]
+    fn bench_addition(bencher: &mut Bencher) {
+        let hash = sha256::Hash::from_hex(
+            "4bf5122f344554c53bde2ebb8cd2b7e3d1600ad631c385a5d7cce23c7785459a",
+        )
+        .unwrap();
+        bencher.iter(|| {
+            let _ = Stump::new().modify(&vec![hash], &vec![], &Proof::default());
+        });
+    }
+    #[bench]
+    fn bench_add_del(bencher: &mut Bencher) {
```

Comment:
Because it only tests against one case? I can add more values if needed.

Reply:
> Because it only tests against one case?

Yeah because of that.


Also when full pollard is working, it might be good to add to a really high tree and delete there? Since for Bitcoin use-cases, the forest will be tall (like 2^28). Probably not the best to add that much for the sake of the benchmark running quick enough but maybe like 10,000 leaves definitely seem good.

It's definitely not easy to do that since you need to hard code the proof in but if full pollard is working, you don't need to do that anymore and can just generate the proof from the pollard.